packages/python-debian/python-debian.spec

```diff
@@ -67,6 +68,11 @@ set -ex
 
 
 %changelog
+* Mon Aug 22 2022 Quirin Pamp <pamp@atix.de> - 0.1.44-2
+- Update to 0.1.44
+- Add zstd dependency to support zstd compression for Ubuntu.
+- Use revision 2 to ensure build against python 3.8.
```

Comment:
This is how I interpreted this comment: https://github.com/theforeman/pulpcore-packaging/pull/548#issuecomment-1219592745

Please double check if my interpretation makes sense/is correct!